src/simmate/calculators/vasp/workflows/base.py

```diff
@@ -117,75 +117,110 @@ class VaspWorkflow(S3Workflow):
     # attribute that is used in the .check() method. and If fix_error=False, I
     # simply print a warning & also add that warning to simmate_corrections.csv
 
-    pre_sanitize_structure: bool = False
+    standardize_structure: str | bool = False
     """
-    In some cases, we may want to sanitize the structure during our setup().
-    This means converting to the LLL-reduced primitive cell. This simply does:
-    ``` python
-    structure_sanitzed = structure.copy(santize=True)
-    ```
+    In some cases, we may want to standardize the structure during our setup().
+    
+    This means running symmetry analysis on the structure in order to reduce
+    the symmetry and also converting it to some standardized form. There
+    are three different forms to choose from and thus 3 different values that
+    `standardize_structure` can be set to:
+    
+    - `primitive`: for the standard primitive unitcell
+    - `conventional`: for the standard conventional unitcell
+    - `primitive-LLL`: for the standard primitive unitcell that is then LLL-reduced
+    - `False`: this is the default and will disable this feature
+    
+    We recommend using `primitive-LLL` when the smallest possible and most cubic
+    unitcell is desired.
+    
+    We recommend using `primitive` when calculating band structures and 
+    ensuring we have a standardized high-symmetry path. Note, existing band
+    structure workflows automatically adjust this value for you.
+    
+    To control the tolerances used to symmetrize the structure, you can use the
+    symmetry_precision and angle_tolerance attributes.
+    
+    By default, no standardization is applied.
     """
 
-    pre_standardize_structure: bool = False
+    symmetry_precision: float = 0.01
     """
-    In some cases, we may want to convert the structure to the standard primitive
-    of a structure. For example, this is required when calculating band structures
-    and ensuring we have a standardized high-symmetry path.
+    If standardize_structure=True, then this is the cutoff value used to determine
+    if the sites are symmetrically equivalent. (in Angstroms)
+    """
+
+    angle_tolerance: float = 5.0
```

Comment:
@jacksund  How was 5 degrees selected?  This seems large to me.  At the end of a multi-stage relaxation, angles should probably be within 0.5 degree of their ideal to be converted.  I checked some of my runs where the "correct" angle was supposed to be 120 degrees.  The maximum deviation was 0.1 degrees.  (I.e., they had angles as bad as 119.9.)

Reply:
It's the default used by `SpacegroupAnalyzer` for pymatgen + spglib. I just copied it over.
https://pymatgen.org/pymatgen.symmetry.analyzer.html#pymatgen.symmetry.analyzer.SpacegroupAnalyzer

Reply:
I'm running a quick test to see how 0.5deg works, and I'll update it if everything looks to be working still

Reply:
Not sure why pymatgen + spglib use such a high default, but switching to 0.5A seems to be fine for our search -- with less risk too. I've changed it to 0.5deg for the `standardize_structure` parameter, but am keep database registration at the pymatgen default.